src/database/sqlite/utils.rs

```diff
@@ -180,6 +180,20 @@ fn migrate_v1_to_v2(conn: &mut rusqlite::Connection) -> Result<(), SqliteDbError
     Ok(())
 }
 
+// After Liana 1.1 we upgraded the schema to add the labels table.
+fn migrate_v2_to_v3(conn: &mut rusqlite::Connection) -> Result<(), SqliteDbError> {
+    db_exec(conn, |tx| {
+        tx.execute(
+            "CREATE TABLE labels (id INTEGER PRIMARY KEY NOT NULL, wallet_id INTEGER NOT NULL, item_kind INTEGER NOT NULL CHECK (item_kind IN (0,1,2)), item TEXT UNIQUE NOT NULL, value TEXT NOT NULL)",
+            rusqlite::params![],
+        )?;
+        tx.execute("UPDATE version SET version = 3", rusqlite::params![])?;
+        Ok(())
+    })?;
+
+    Ok(())
+}
```

Comment:
Would be nice to test the migration, even if it's a trivial unit test that we can now store labels at the end of the existing migration unit test.